ghastly/ghastly.py

```diff
@@ -41,16 +41,26 @@ def fill_core(input_file, rough_pf):
     sim_block = input_block.create_obj()
 
     rough_pack = []
-    pack_zones = sim_block.core_main | sim_block.core_outtake
-    core_volume = sum([i.volume for i in pack_zones.values()])
+    match sim_block.down_flow:
+        case True:
+            pack_zones = sim_block.core_main | sim_block.core_outtake
+        case False:
+            pack_zones = sim_block.core_main | sim_block.core_intake
+
     for element in pack_zones.values():
         if type(element) == ghastly.core.CylCore:
             coords = pack_cyl(sim_block, element, rough_pf)
             rough_pack += coords
         else:
             pass
 
-    n_pebbles = int((sim_block.pf*core_volume)/sim_block.pebble_volume)
+    # because pebbles will settle into the outtake region, the target number
+    # of pebbles is calculated using an assumed 0.6 pf in the outtake region,
+    # and the target pf in the main region
```

Comment:
This is a comment for another time. Could there be a scenario where someone wants to control the packing fraction of the outtake section? Maybe even over time if they're trying to make an accident scenario of some sort. Not something to do now, just something to think about